### PR TITLE
Add contribution guide

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- Work in feature branches, and make sure to base your work on the appropriate base branch. If you started from the wrong branch, please rebase and change the PR target.
+- Use our [pre-commit](#pre-commit) hooks.
+- Write [meaningful commit messages](https://cbea.ms/git-commit/) and use [conventional commit messages](#conventional-commits).
+- Open a [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) to get early feedback.
+- Ensure that every commit builds. Please fix broken commits by rebasing.
+- Run `cargo test` before committing to ensure that all tests pass.
+- Try to split up bigger changes into smaller PRs that are easier to review (which means they will be merged faster!)
+
+
+## Using pre-commit Hooks <a name="pre-commit"></a>
```
Comment (Swiftb0y) [inline CONTRIBUTING.md:16]:
```suggestion
## Using pre-commit Hooks
```
I don't think explicit HTML anchors are needed...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  | `build`      | Affects the build system or external dependencies.
+  | `chore`      | Does not belong into any other category and should not be listed in the changelog.
+  | `ci`         | Changes the CI/pre-commit configuration files and scripts.
+  | `docs`       | Only changes the documentation.
+  | `feat`       | Adds a new feature.
+  | `fix`        | Fixes a bug present in a released version (otherwise use `chore`).
+  | `perf`       | Improves the performance.
+  | `refactor`   | Changes the code but neither fixes a bug nor adds a feature.
+  | `style`      | Only reformats the code (white-space, formatting, missing semi-colons, etc)
+  | `test`       | Affects unit/integration tests.
```
Comment (Swiftb0y) [inline CONTRIBUTING.md:44]: I'd add a closing sentence. Something like "Thank you very much for your interest in rekordcrate, we're looking forward to your pull requests."

**Review by Swiftb0y — COMMENTED** [review]
Thank you very much for taking care of this. Its written very well!

**Review by Swiftb0y — DISMISSED** [review]
Thank you very much

**Comment by Holzhaus** [comment]
@Swiftb0y The table formatting was broken, I fixed it and force-pushed.

**Comment by Swiftb0y** [comment]
Sorry I didn't catch that. :+1:

**Pull request merged**